### DISCUSSION TITLE
why is it impossible to join github education?

### DISCUSSION THREAD

**Discussion by Lavin3006** [community/community#146260] [GitHub Education]
### Hi everyone,

i have uploaded my student id card multiple times, where the university logo is to be seen as well as the expiration date. another issue is that all devices only allow to take a picture with a front camera and this is problematic. my name on my billing details matches the name on the student id yet i keep getting rejected. Support is also useless. has anyone had any luck?

**Comment by tony9823** [comment]
Has anyone successfully verified their student ID [here](https://www.jenny-modpro.com/) despite repeated rejections? I’ve uploaded clear photos showing the university logo and expiration date, matching my billing details. The front-camera-only restriction complicates things, and support hasn’t helped. Any advice or similar experiences?

**Comment by EchoSingh** [comment]
**Try Adding Your University Email to GitHub:**
   - A verified university email (e.g., `name@university.edu`) can sometimes help in confirming your student status.
   - Go to **GitHub Settings > Emails**, then add your university email and verify it.

**Comment by tony9823** [comment]
Has anyone successfully resolved a situation like this? I've repeatedly uploaded my student ID card showing [here](https://etihadtownsialkot.com/) the university logo and expiration date, but it keeps getting rejected. The front-camera-only feature complicates this, even though my billing name matches the ID. Support hasn't helped—any advice or similar experiences?

**Comment by Sheriffmustapha** [comment]
> ### Hi everyone,
> 
> i have uploaded my student id card multiple times, where the university logo is to be seen as well as the expiration date. another issue is that all devices only allow to take a picture with a front camera and this is problematic. my name on my billing details matches the name on the student id yet i keep getting rejected. Support is also useless. has anyone had any luck?

**Comment by james9832** [comment]
Why is it seemingly impossible to join GitHub [here](https://lahorefuturecity.com/) Education even after meeting all the eligibility criteria, such as being a verified student with a valid institutional email, and how can one overcome frequent rejections or verification issues effectively?

**Comment by james9832** [comment]
Its Easy Grader and Final Grade Calculator turn scores into accurate percentages and letter grades in seconds. A must-have tool for fast, reliable academic assessment. [Click here](https://calculategrader.com/) for more information.

**Comment by alinkmidest-tech** [comment]
GitHub Education’a katılmak bazen imkânsız gibi görünebilir çünkü başvuru süreci oldukça sıkı doğrulamalara dayanır. Öğrenci kimliğinin net olmaması, okul e-posta adresinin bulunmaması, belgelerde isim uyuşmazlığı veya güncel öğrenci durumunun kanıtlanamaması başvurunun reddedilmesine neden olur. Ayrıca sadece tanınmış eğitim kurumları kabul edilir. Bu durum, bazı dijital platformlara erişimde yaşanan zorluklara benzer; örneğin [inat tv giriş](https://inat-tv.com.tr/) sırasında da kullanıcılar doğrulama veya erişim problemleri yaşayabilir. Her iki sistem de güvenlik ve doğruluk için katı kurallar uygular.
